Future/inst/shiny/meal app/app_for_meal/app.R

```diff
@@ -0,0 +1,283 @@
+                          )),
+                   shinyWidgets::actionBttn(
+                       inputId = "add01",
+                       label = "Dodaj produkt",
+                       size = "sm",
+                       style = "jelly",
+                       color = "primary"
+                   )
+               )),
+               hidden(div(id = "box02", box(
+                   width = 12,
+                   column(
+                       6,
+                       pickerInput(
+                           inputId = "product02",
+                           label = "Wybierz produkt",
+                           choices = "",
+                           options = list(`live-search` = TRUE)
+                       )
+                   ),
+                   column(
+                       4,
+                       numericInput(
+                           inputId = "weight02",
+                           label = "Wpisz gramature produktu",
+                           value = 0
+                       )
+                   ),
+                   shinyWidgets::actionBttn(
+                       inputId = "add02",
+                       label = "Dodaj produkt",
+                       size = "sm",
+                       style = "jelly",
+                       color = "primary"
+                   ),
+                   shinyWidgets::actionBttn(
+                       inputId = "remove02",
+                       label = "Usun produkt",
+                       size = "sm",
+                       style = "jelly",
+                       color = "danger"
+                   ),
+               ))),
+               hidden(div(id = "box03", box(
+                   width = 12,
+                   column(
+                       6,
+                       pickerInput(
+                           inputId = "product03",
+                           label = "Wybierz produkt",
+                           choices = "",
+                           options = list(`live-search` = TRUE)
+                       )
+                   ),
+                   column(
+                       4,
+                       numericInput(
+                           inputId = "weight03",
+                           label = "Wpisz gramature produktu",
+                           value = 0
+                       )
+                   ),
+                   shinyWidgets::actionBttn(
+                       inputId = "add03",
+                       label = "Dodaj produkt",
+                       size = "sm",
+                       style = "jelly",
+                       color = "primary"
+                   ),
+                   shinyWidgets::actionBttn(
+                       inputId = "remove03",
+                       label = "Usun produkt",
+                       size = "sm",
+                       style = "jelly",
+                       color = "danger"
+                   ),
+               ))),
+               hidden(div(
```

Comment:
be consistent - use one style:  
`hidden(div(box` 
or
`hidden(div(
box(`

Reply:
done